pkg/roachprod/prometheus/prometheus.go

```diff
@@ -341,12 +341,13 @@ sudo systemd-run --unit prometheus --same-dir \
 			l.Stderr, cfg.PrometheusNode, "install grafana",
 			fmt.Sprintf(`
 sudo apt-get install -qqy apt-transport-https &&
-sudo apt-get install -qqy software-properties-common wget &&
+sudo apt-get install -qqy software-properties-common &&
 sudo apt-get install -y adduser libfontconfig1 &&
-wget https://dl.grafana.com/enterprise/release/grafana-enterprise_9.2.3_%s.deb -O grafana-enterprise_9.2.3_%s.deb &&
-sudo dpkg -i grafana-enterprise_9.2.3_%s.deb &&
+echo "Downloading https://dl.grafana.com/enterprise/release/grafana-enterprise_9.2.3_%[1]s.deb" &&
+curl https://dl.grafana.com/enterprise/release/grafana-enterprise_9.2.3_%[1]s.deb -sS -o grafana-enterprise_9.2.3_%[1]s.deb &&
```

Comment:
You may want -f to get errors on non-2xx responses and -L to follow redirects.

Reply:
According to my man page, -f means "Fail fast with no output at all on server errors"